packages/integration-tests/helpers/childChainHelper.js

```diff
@@ -16,9 +16,15 @@ limitations under the License. */
 const promiseRetry = require('promise-retry')
 const { transaction, hexPrefix } = require('@omisego/omg-js-util')
 const numberToBN = require('number-to-bn')
+const Childchain = require('@omisego/omg-js-childchain')
+const config = require('../test-config')
 
-function selectUtxos (utxos, amount, currency, includeFee) {
+async function selectUtxos (utxos, amount, currency) {
   // Filter by desired currency and sort in descending order
+  const childChain = new Childchain({ watcherUrl: config.watcher_url, watcherProxyUrl: config.watcher_proxy_url })
+  const fees = (await childChain.getFeesInfo())['1']
```

Comment:
yea just curious but do you know why there is "1" in the response??? will there be a "2"? lol

Reply:
@nicholasmueller 1 is the transaction type, the fee is based on that.